.env-gh-codespaces-sample

```diff
@@ -0,0 +1,36 @@
+# This configuration adds the following overlays:
+#
+# docker-gh-codespaces: GitHub Codespaces settings (eg port 3000 exposed)
+#
+# docker-compose-publish-capybara-port.yml: expose the capybara port so a chrome driver
+#   running on the host machine can talk to the capaybara server and you can see the UI
+#   while the test are running.
+#
+# docker/dev/docker-compose-graphql.yml: start a graphql back-end server and
+#   a react-admin interface for managing administrative tasks
+COMPOSE_FILE=docker-compose.yml:docker/dev/docker-compose-gh-codespaces.yml:docker/dev/docker-compose-publish-capybara-port.yml:docker/dev/docker-compose-graphql.yml:docker/dev/docker-compose-lara-proxy.yml
+
+# The URL for ourselves. This is used when resources are published to the portal.
+# SITE_URL in docker-compose will be constructed from PORTAL_HOST and PORTAL_PROTOCOL.
+# The PORTAL_HOST variable is also used to set VIRTUAL_HOST variable. GitHub Codespaces will assign a random
+# host name that can be obtained after the port is made public in the Visual Studio Code "Ports" tab.
+#
+# For automation PORTAL_HOST should be learn.dev.docker and PORTAL_PROTOCOL should be https.
+PORTAL_HOST=[PORTAL-RANDOM-GH-CODESPACES-HOST]].githubpreview.dev
```

Comment:
Did you mean to have 2 `]]` at the end?

Reply:
Thanks, this is gone in the upcoming PR, so I'm going to skip fixing it here to avoid merge conflicts.